core/templates/dev/head/pages/topic_editor/main_editor/TopicEditorTabDirective.js

```diff
@@ -45,6 +45,8 @@ oppia.directive('topicMainEditor', [
           $scope.$on(EVENT_TOPIC_INITIALIZED, _initEditor);
           $scope.$on(EVENT_TOPIC_REINITIALIZED, _initEditor);
 
+          _initEditor();
```

Comment:
As a general rule, let's maybe try to put all the function declarations, scope bindings, etc. at the top and the "main" code at the bottom. I think we do too much sprinkling of the main code all over the controller file, in general.

/cc @apb7 @kevinlee12

Reply:
Ok, done.